pkg/apis/discovery/types.go

```diff
@@ -57,21 +57,26 @@ type EndpointSlice struct {
 type AddressType string
 
 const (
-	// AddressTypeIP represents an IP Address. Inclusive of IPv4 and IPv6
-	// addresses.
+	// AddressTypeIP represents an IP Address.
+	// This address type has been deprecated and has been replaced by the IPv4
+	// and IPv6 adddress types. New resources with this address type will be
+	// considered invalid. This will be fully removed in 1.18.
+	// +deprecated
```

Comment:
Let what `api-reviewers` will comeback with. on deprecating in ALPHA state apis

Reply:
I prefer to deprecate if we can afford to do so.

Reply:
as long as we don't allow the deprecated value to be created or added via the beta EndpointSlice API, marking as deprecated in alpha is fine

Reply:
thanks for clarifying!